bin/make_packages_yaml

```diff
@@ -53,11 +61,16 @@ Linux)
         if [ x$RPMCACHE != x ]
         then
         getv() {
-           grep "^$1-[0-9]" $RPMCACHE | tail -1 | grep -v 'is not installed' | sed -e 's/[^-]*-//' -e 's/[a-z][^-]*-//' -e 's/-.*//' || runversion $1
+           grep "^$1-[0-9]" $RPMCACHE | tail -1 | grep -v 'is not installed' | 
+perl-Data-Dumper-2.145-3.el7.x86_64
+              sed -e 's/.*-\([0-9][0-9.kp]*\)-[0-9][0-9.]*\..*/\1/' ||
+              runversion $1
```

Comment:
Not sure what's going on here?

Reply:
`perl-DataDumper-...` is a package spec, but you're treating it as a command and then breaking the pipe chain? Looks like an erroneous paste...

Reply:
This is what I get for trying to resolve merges in the github GUI... sigh.

Reply:
fixed.